lab_1/main.py

```diff
@@ -49,9 +96,21 @@ def get_top_n_words(freq_dict: dict, top_n: int) -> list:
     top_n = 1
     --> ['happy']
     """
-    pass
+    top_n_words = []
+    if type(freq_dict) == dict and type(top_n) == int:
```

Comment:
Usually, there are check for input arguments and body function - on the same level.
No need for nesting.